tests-example/tilt_modules/tests

```diff
@@ -0,0 +1 @@
+/Users/maia/code/go/src/github.com/tilt-dev/tilt-extensions/tests
```

Comment:
this also seems to be a symlink, i don't think this will work properly if you check it in

Reply:
whoops, that's what I had hooked up for developing -- will delete.

Reply:
:+1: